README.md

```diff
@@ -119,3 +119,9 @@ You can either call modules, rulebooks and playbooks by their Fully Qualified Co
 ## License Information
 
 See [LICENSE](./LICENSE) to see the full text.
+
+## Generating the api client
+
+1. Install the [openapi-python-client](https://github.com/openapi-generators/openapi-python-client) library
```

Comment:
question: is the version of the generator something that needs to be pinned?

Reply:
The usage of "openapi-python-client" is optional - only in the cases we need to regenerate the client files. Other then that, you can use the collection without it.
Should we add it to the requirements.txt file or just have installation instructions?
My answer to that is we don't need to increase the dependencies as it's not required during runtime of the collection. Only a collection developer sometimes may need it (when the API changes).
When updating the generated client, I think we should try to use the latest generator release, even if it's different from the last one used.